source/release-notes.txt

```diff
@@ -10,6 +10,58 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.34.0
+----------------
+
+New Features:
+
+- Add command-line interface and global configuration (:issue:`COMPASS-6069`, 
+  :issue:`COMPASS-6070`, :issue:`COMPASS-6071`, :issue:`COMPASS-6073`, 
+  and :issue:`COMPASS-6074`)
+
+- Flip the new toolbars feature flag, always show new toolbars 
+  (:issue:`COMPASS-5679`)
+
+- Add autocompleter for aggregation, use autocompleter in import pipeline 
+  modal (:issue:`COMPASS-6175`)
+
+- Add ``protectConnectionStrings`` option (:issue:`COMPASS-6066`)
+
+- Implement ``HADRON_ISOLATED`` through preferences (:issue:`COMPASS-6065`)
```

Comment:
The linked ticket COMPASS-6065  is talking about the `networkTraffic` config option, which seems to be the user-facing portion of this update. Maybe we can tweak this to say something like:

- Add `networkTraffic` configuration option to block outgoing network connections.

Reply:
Fixed